.travis.yml

```diff
@@ -35,7 +35,9 @@ before_install:
   - if [[ -n $CC ]]; then $CC --version; fi
 install:
   - pip install "poetry<2,>=1.0" tox
+  # These are build requirements. poetry does not offer a way to specify build requirements.
   - pip install "mypy-protobuf==1.21"
+  - pip install "Cython<1,>=0.29"
```

Comment:
Is this defined here or inside poetry?

Reply:
It's a build requirement. We need it to be installed before poetry calls build.py. There's no way to do this in poetry.